ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -22,12 +22,10 @@ export class LoggingService {
 
     public logError: Logger = this.showEvent(LogLevel.Error) ?
         (...args) => {
-            const stringifiedArgs = args.map(item => (
-                 (typeof item === 'object') ? this.stringify(item) : item
-            ));
-
-            this.stackdriverErrorReporterService.handleError(stringifiedArgs.join(', '));
-            console.error.apply(window.console, stringifiedArgs);
+            args.forEach(arg => {
+                console.error.apply(window.console, arg);
+                this.stackdriverErrorReporterService.handleError(arg);
+            });
```

Comment:
Old way for stringify is not working if loop `item` does not have a `[key: value]` pair.

For example, in the screenshot below,  on the right side in debugger Scope, `stringifiedArgs` is the return value. But it does not contains the the error message anymore.

![logError](https://github.com/broadinstitute/ddp-angular/assets/35533885/b6f6ebeb-9108-4378-abbc-2f33d213ae57)

As the result, exception logs no longer contain the original error message.
```
Error: Logging, HTTP POST: https://us-central1-broad-ddp-dev.cloudfunctions.net/LoggingService. Error:,
```
(note: ticket for above error is PEPPER-1012. Cause is still unknown to me because I am  unable to reproduce it. But I think it's related to expired session so I added code to log that)

<img width="1598" alt="Screenshot 2023-08-18 at 11 11 08 AM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/eb2e98c0-9f12-4d86-ab3c-a633c528dfee">

Reply:
Before proceeding with these modifications, let's delve into scenarios where it might not function as intended and have a discussion about them